synapse/storage/databases/main/event_federation.py

```diff
@@ -730,17 +730,13 @@ async def get_backfill_points_in_room(
         limit: int,
     ) -> List[Tuple[str, int]]:
         """
-        Gets the oldest events(backwards extremities) in the room along with the
-        approximate depth. Sorted by depth, highest to lowest (descending) so the closest
-        events to the `current_depth` are first in the list.
+        Get the backward extremities to backfill from in the room along with the
+        approximate depth. Only returns events that are at a depth lower than or
+        equal to the `current_depth`. Sorted by depth, highest to lowest (descending)
+        so the closest events to the `current_depth` are first in the list.
```

Comment:
convention is to start with a one line description, rather than a paragraph:

```suggestion
        Get the backward extremities to backfill from in the room.
        
        Only returns events that are at a depth lower than or
        equal to the `current_depth`. Sorted by depth, highest to lowest (descending)
        so the closest events to the `current_depth` are first in the list.
```

... but that's really gold-plating.